src/components/Navbar/Navbar.scss

```diff
@@ -0,0 +1,33 @@
+@use '../../styles/variables' as var;
+@use '../../styles/mixins' as mix;
+
+.nav {
+  position: sticky;
+  top: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: var.$primary-color;
+  padding: 1rem 2rem;
+
+  ul {
+    display: flex;
+    gap: 1.25rem;
+    list-style: none;
+    align-items: center;
+    justify-content: flex-end;
+  }
+
+  a {
+    color: inherit;
+    text-decoration: none;
+  }
+  li {
```

Comment:
This is semi-fine; you are targeting specific elements, which is better than something like `> * {}.`. It is quite flat, but regardless, avoid it. Always prefer using classnames especially when you OWN all components, which are literally in the same file. 

This should be used as a last resort when there is no other way to target the elements. The problem with it is the fact that other developers will NOT expect it. It is not apparent from the JSX. It can also be declared multiple levels higher. You come to your lovely safe space component. You add `<li>` to it. And suddenly, you get random styles to it :)